### DISCUSSION TITLE
Zoom

### DISCUSSION THREAD

**Discussion by AliKerimov** [joyceworks/flowchart-react#35] [Q&A]
Hi, your work is just amazing, but there is some problem with zoom diagram it works only 60% and it did not work below this number, is there any solution for this?  and also when scroll with mouse on svg in console it returns this error: Unable to preventDefault inside passive event listener invocation.

**Comment by iamppz** [comment]
Released version 3.10.5 which now allows the minimum zoom scale to be set to 0.1.
